src/data/zwave_js.ts

```diff
@@ -284,14 +284,24 @@ export const migrateZwave = (
 
 export const fetchZwaveNetworkStatus = (
   hass: HomeAssistant,
-  device_or_entry_id: string,
-  id_type: "entry_id" | "device_id" = "entry_id"
-): Promise<ZWaveJSNetwork> =>
-  hass.callWS({
+  device_or_entry_id: {
+    device_id?: string;
+    entry_id?: string;
+  }
+): Promise<ZWaveJSNetwork> => {
+  if (device_or_entry_id.device_id && device_or_entry_id.entry_id) {
+    throw new Error("Only one of device or entry ID should be supplied.");
+  }
+  if (!device_or_entry_id.device_id && !device_or_entry_id.entry_id) {
+    throw new Error("Either device or entry ID should be supplied.");
+  }
+  return hass.callWS({
     type: "zwave_js/network_status",
-    device_or_entry_id,
-    id_type,
+    device_or_entry_id:
+      device_or_entry_id.device_id || device_or_entry_id.entry_id,
+    id_type: device_or_entry_id.device_id ? "device_id" : "entry_id",
```

Comment:
Would you not also want to update this on the backend? It is fine by my either way :-)

Reply:
ah I guess we can do this a little less complicated. I will fix